experiments/Train.ipynb

```diff
@@ -21,6 +21,35 @@
     "    * [Predict with serving model](#Predict-with-serving-model)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1010d9ca",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "  import google.colab\n",
+    "  IN_COLAB = True\n",
+    "except:\n",
+    "  IN_COLAB = False\n",
+    "# eventual initialization for collab notebooks\n",
+    "assert IN_COLAB and not os.path.exists(\".git\"), \"Nothing to do\"\n",
```

Comment:
Will notebook continue to execute in "run all" (and similar) mode if assert raises here?